cli/pack/docker.go

```diff
@@ -31,9 +33,21 @@ func PackInDocker(cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx,
 	envDir := filepath.Dir(cmdCtx.Cli.ConfigPath)
 
 	goTextEngine := templates.NewDefaultEngine()
+
+	tntVersion := cmdCtx.Cli.TarantoolVersion
+
+	tntVerParsed, err := version.Parse(packCtx.TarantoolVersion)
+	if err == nil {
```

Comment:
We silently use current tarantool version if the specified version format is incorrect. I think this is unexpected. We must fail if we are unable to use the version that was specified by user.
Need a test for this.

Reply:
Updated.